tests/test_run_assemble.py

```diff
@@ -47,6 +47,28 @@ def test_main(self, mock_copy, mock_temp, mock_recorder, mock_bundles, *mocks):
         mock_bundle.install_min.assert_called()
         mock_bundle.install_plugins.assert_called()
 
-        mock_bundle.build_tar.assert_called_with("curdir/bundle")
+        mock_bundle.build_tar.assert_called_with("curdir/dist")
 
-        mock_recorder.return_value.write_manifest.assert_has_calls([call("path"), call("curdir/bundle")])  # manifest included in tar
+        mock_recorder.return_value.write_manifest.assert_has_calls([call("path"), call("curdir/dist")])  # manifest included in tar
+
+    @patch("os.chdir")
+    @patch("os.makedirs")
+    @patch("os.getcwd", return_value="curdir")
+    @patch("argparse._sys.argv", ["run_assemble.py", BUILD_MANIFEST, "--base-url", "baseurl"])
+    @patch("run_assemble.Bundles", return_value=MagicMock())
+    @patch("run_assemble.BundleRecorder", return_value=MagicMock())
+    @patch("run_assemble.TemporaryDirectory")
+    @patch("shutil.copy2")
+    def test_url(self, mock_copy, mock_temp, mock_recorder, mock_bundles, *mocks):
```

Comment:
This test doesn't actually seem to test anything related to `base-url`?

Reply:
I just mean that this test doesn't really test anything different than the one above, or does it?

If you're adding a test that covers the end-to-end scenario with a different base_url, it should expect something different. Either add that or just remove the test?